src/main/resources/interface-specification.yml

```
@@ -8820,6 +8820,79 @@ paths:
                 schema:
                   type: integer
                 description: Time interval in milliseconds. Allowed requests will be constantly replenished during the interval. At the end of the interval the max allowed requests will be available
+  /privacyNotices/{consentType}:
+    parameters:
+        - $ref: '#/components/parameters/CorrelationIdHeader'
+        - $ref: '#/components/parameters/IpAddress'
+        - name: consentType
+          in: path
+          required: true
+          schema:
+            $ref: '#/components/schemas/ConsentType'
+    get:
+        tags:
+          - privacyNotices
+        operationId: getPrivacyNoticeFromBucket
+        description: Retrieve a specified version of privacy notice from AWS bucket
```

No need to expose that kind of info in the openapi spec
```suggestion
        description: Retrieve the content of the privacy notice version
```